### DISCUSSION TITLE
Is it able to sort columns while moving like the effect of react-dnd?

### DISCUSSION THREAD

**Discussion by LvChengbin** [KevinVandy/material-react-table#371] [Q&A]
The UX of current dnd is not smooth, if I want to implement the effect like react-dnd, what shall I do?

Here is the demo of react-dnd. https://react-dnd.github.io/react-dnd/examples/sortable/simple

**Comment by KevinVandy** [comment]
No, but if someone contributed some lightweight performant CSS that accomplished that, I would not be against it.

**Reply by LvChengbin** [reply]
I mean not only the UI style but also the interactive, I prefer to sort items immediately while dragging over another item instead of sorting them after I drop the element.
I am glad to implement this by myself, but I found it may not easy to change the default dnd behavior of `mrt`.
Do you have any suggestions? for example which API or hooks I can use to try doing that?
